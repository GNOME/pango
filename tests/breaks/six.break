a file:///<span segment="word">ho/</span><span segment="word">bo-mo/</span>Bla-txt file
