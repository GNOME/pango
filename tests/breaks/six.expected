Text:         ⁦a⁩  [ ]  ⁦f⁩ ⁦i⁩ ⁦l⁩ ⁦e⁩  ⁦:⁩ ⁦/⁩ ⁦/⁩ ⁦/⁩  ⁦h⁩ ⁦o⁩ ⁦/⁩   ⁦b⁩ ⁦o⁩ ⁦-⁩ ⁦m⁩ ⁦o⁩ ⁦/⁩   ⁦B⁩ ⁦l⁩ ⁦a⁩  ⁦-⁩  ⁦t⁩ ⁦x⁩ ⁦t⁩  [ ]  ⁦f⁩ ⁦i⁩ ⁦l⁩ ⁦e⁩  [0x0a]  
Breaks:     c  c    lc c c c c  c c c lc c c lc  c c c c c lc  c c c  lc c c c    lc c c c c       Lc
Whitespace:    x                                                             x             w       w 
Sentences:  bs                                                                             e       b 
Words:      bs be   bs       be b b b bs     bse           bse     be bs     be   bs       be      b 
Graphemes:  b  b    b  b b b b  b b b b  b b b   b b b b b b   b b b  b  b b b    b  b b b b       b 
Hyphens:               i i i             i       i     i       i i       i i         i i i           
